display_list/dl_vertices.cc

```diff
@@ -267,6 +267,15 @@ void DlVertices::Builder::store_colors(const DlColor colors[]) {
   needs_colors_ = false;
 }
 
+void DlVertices::Builder::store_colors(const uint32_t colors[]) {
```

Comment:
This is a potentially expensive operation. Since we're storing the colors anyway, can we change store colors to some kind of reserve and then one at a time write?

Reply:
Hopefully this will be deprecated and/or only used in the skia backend.  It wasn't a big deal though so I made it faster by writing into the pod directly.